src/order/hom/complete_lattice.lean

```diff
@@ -626,3 +626,22 @@ See also `complete_lattice_hom.set_preimage`. -/
   right_inv := λ s, by simp only [← image_comp, equiv.self_comp_symm, id.def, image_id'],
   map_rel_iff' :=
     λ s t, ⟨λ h, by simpa using @monotone_image _ _ e.symm _ _ h, λ h, monotone_image h⟩ }
+
+section
+
+open set
+
+/-- The map `(a, b) ↦ a ⊓ b` as an `Inf_hom`. -/
+@[simps] def inf_Inf_hom (α) [complete_lattice α] : Inf_hom (α × α) α :=
+{ to_fun := λ x, x.1 ⊓ x.2,
+  map_Inf' := λ s, begin
+    -- TODO: use lemmas from gh-18029
+    change Inf (_ '' _) ⊓ Inf (_ '' _) = _,
+    simp_rw [Inf_image, infi_inf_eq],
+  end }
```

Comment:
Assuming your comment from earlier works,
```suggestion
  map_Inf' := λ s, by simp_rw [fst_Inf, snd_Inf, Inf_image, infi_inf_eq] }
```
You might need a `prod.` prefix somewhere.

Reply:
Yes. My original draft had `open prod` higher up, but I've pushed `simp_rw [prod.fst_Inf, prod.snd_Inf, Inf_image, infi_inf_eq]`